packages/compass-schema/src/stores/schema-export-reducer.ts

```diff
@@ -238,7 +238,9 @@ export const changeExportSchemaFormat = (
     try {
       const schemaAccessor = schemaAccessorRef.current;
       if (!schemaAccessor) {
-        throw new Error('No schema analysis available');
+        throw new Error(
+          "No schema analysis available. Please analyze the collection's schema before exporting."
```

Comment:
Drive-by, updated this error message to be a bit more descriptive as folks do see it if they try exporting and they have not yet run analysis.